README.md

```diff
@@ -63,19 +63,17 @@ See https://github.com/settings/tokens to set up personal access tokens.
 
 ## Requirements
 
- - Ruby (tested against v2.3 and above)
+ - Ruby (tested against v2.5 and above)
  - GitHub access (see Configuration section)
 
-## Contact ![Join the chat at https://gitter.im/envato/unwrappr](https://badges.gitter.im/Join%20Chat.svg)
+## Contact
 
  - [GitHub project](https://github.com/envato/unwrappr)
- - [Gitter chat room](https://gitter.im/envato/unwrappr)
  - Bug reports and feature requests are welcome via [GitHub Issues](https://github.com/envato/unwrappr/issues)
 
 ## Maintainers
 
  - [Pete Johns](https://github.com/johnsyweb)
- - [Joe Sustaric](https://github.com/joesustaric)
```

Comment:
😢

Reply:
I mean I'm happy to still be a maintainer..  But I don't know what Envatos policy is RE: alumni and OS repos in their org.

Reply:
If you're happy, I'm happy! 😀